### PR TITLE
correct failure due to timezone handling

### PR DESCRIPTION
**Pull request by BFalquet**
correct error in integration tests.

In the listings, the time reported should be the time of the event at the location where the event was recorded. Using base function changes the result depending on the time zone of the computer. Use `lubridate` to circumvent the issue. 

thank you for the review.

### CODE REVIEW

**Review by clarkliming — APPROVED** [review]
interesting issue! I did not know that the timezone would impact the date

File: R/utils.R [hunk 1]
```diff
@@ -438,11 +438,23 @@ listing_format_chevron <- function() {
...
 #' @return a `function` converting a date into `string`.
 #'
-#' @export
+#' @note The date is extracted at the location of the measure, not at the location of the system.
 #'
+#' @export
+#' @examples
+#' format_date("%d%b%Y")(as.Date("2021-01-01"))
+#' format_date("%d%b%Y")(as.POSIXct("2021-01-01 00:00:01", tz = "NZ"))
+#' format_date("%d%b%Y")(as.POSIXct("2021-01-01 00:00:01", tz = "US/Pacific"))
```
Comment (clarkliming) [inline R/utils.R:447]: while this is a nice check, but it reminds me if we should use the POSIXct in the listing, or use the variable that is date only?

Comment (BFalquet) [reply R/utils.R:447]: I am not sure, the current data set is in POSIXct so I assumed it was the correct format. I works with both, but for the POSIXct we have to be a bit careful and make sure that the time zone is set.

Comment (clarkliming) [reply R/utils.R:447]: and how about using the TRTSDT instead of TRTSDTM here as the default? (of course this update is a nice one can be kept)

Comment (BFalquet) [reply R/utils.R:447]: I created an issue here: https://github.com/insightsengineering/chevron/issues/752

**Pull request merged**